plaso/parsers/mcafeeav.py

```diff
@@ -137,8 +139,10 @@ def VerifyRow(self, parser_mediator, row):
     # This file can have a UTF-8 byte-order-marker at the beginning of
     # the first row.
     # TODO: Find out all the code pages this can have.  Asked McAfee 10/31.
-    if row['date'][0:3] == b'\xef\xbb\xbf':
-      row['date'] = row['date'][3:]
+    byte_order_mark = codecs.decode(b'\xef\xbb\xbf', parser_mediator.codepage)
```

Comment:
please do not change b'\xef\xbb\xbf' is a sequence specific to UTF-8 (UTF-8 BOM, which technically does not need a BOM ;), this is the binary sequence and should not be translated into a string. Seeing that the string u'\ufeff' can be used for UTF-16 and UTF-32 as well.